spec/aep-64/README.md

```diff
@@ -0,0 +1,76 @@
+---
+aep: 64
+title: "JWT authentication for provider API"
+author: Artur Troian (@troian)
+status: Draft
+type: Standard
+category: Core
+created: 2025-04-03
+updated: 2025-04-03
+roadmap: major
+---
+
+## Motivation
+
+Enhance tenant authentication to the provider gateway using JWT allows client API to retain communication with leases during blockchain maintenance.
+
+## Summary
+
+mTLS authentication is great security mechanism, however, with current implementation when blockchain acts as root of trust, it becomes complicated for API clients to retain communication with leases during blockchain maintenance windows. There are also other limitations like granual access. Certificate gives access to all all leases and all features, tho granual access is possible via certificate, implementing and maintaining it for clients is challenging topic. JWT on other hand is wide adopted approach for authentication and authorization.
```

Comment:
```suggestion
mTLS authentication is great security mechanism, however, with current implementation when blockchain acts as root of trust, it becomes complicated for API clients to retain communication with leases during blockchain maintenance windows. There are also other limitations like granual access. Certificate gives access to all leases and all features, although granual access is possible via certificate, implementing and maintaining it for clients is a challenging topic. JWT on other hand is wide adopted approach for authentication and authorization.
```

Reply:
It's also very challenging for client applications to manage certificates as the client need to store the private key somewhere (in the browser local storage in the case of console and up to the customer for the managed wallet api)